Use the OCI Helm Charts repository instead of the manually maintained Helm Chart repo on the website (#8791)
strimzi/strimzi-kafka-operator: documentation/modules/managing/proc-drain-cleaner-deploying-helm-chart.adoc

.Procedure

. Deploy the Drain Cleaner:
+
[source,shell]
----
helm install strimzi-drain-cleaner oci://quay.io/strimzi-helm/strimzi-drain-cleaner
----
+

> Alternatively, you can also install an exact version of the Drain Cleaner or specify any changes to the default configuration as parameter values.

Reviewer: ```suggestion
Alternatively, you can use parameter values to install a specific version of the Drain Cleaner or specify any changes to the default configuration.
```
